src/components/_global/BalBtn/BalBtn.vue

```diff
@@ -37,6 +40,10 @@ export default defineComponent({
       validator: (val: string): boolean =>
         ['button', 'a', 'div', 'router-link'].includes(val)
     },
+    to: {
+      type: Object as PropType<{ name: string }>,
+      default: null
+    },
```

Comment:
This isn't needed in the original version because props on BalBtn components apply directly to the first element in the component. If we do absolutely need a wrapper component we should probably add `v-bind="attrs"` to the `<component ...` element.